### PR TITLE
Enable concurrent lumisection processing in ECAL

### PR DESCRIPTION
**Pull request by tanmaymudholkar**
Changed ECAL Monitor Tasks from `DQMOneLumiEDAnalyzer` to `DQMOneEDAnalyzer`. All by-lumisection plots are disabled for the time being.

#### PR description:

There has been a long-standing request to enable concurrent processing for the ECAL DQM workflows. The problematic functions in `DQMOneLumiEDAnalyzer` are the ones that act at the beginning and end of the lumisection transitions. This PR migrates to `DQMOneEDAnalyzer` and disables all by-lumisection plots for now.

#### PR validation:

We have run the recommended suite of tests: `runTheMatrix.py -l limited -i all --ibeos`

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a backport.

### CODE REVIEW

File: DQM/EcalMonitorClient/src/SummaryClient.cc [hunk 1]
```diff
@@ -118,14 +106,14 @@ namespace ecaldqm {
       int trigprim(sTriggerPrimitives ? (int)sTriggerPrimitives->getBinContent(id) : kUnknown);
       int rawdata(sRawData.getBinContent(id));
 
-      double rawdataLS(sFEStatusErrMapByLumi.getBinContent(id));  // Includes FE=Disabled
+      // double rawdataLS(sFEStatusErrMapByLumi.getBinContent(id));  // Includes FE=Disabled
```
Comment (jfernan2) [inline DQM/EcalMonitorClient/src/SummaryClient.cc:109]: Better removed if not used

File: DQM/EcalMonitorClient/src/TrigPrimClient.cc [hunk 2]
```diff
@@ -112,33 +106,18 @@ namespace ecaldqm {
     // Loop over all TTs
     for (unsigned iTT(0); iTT < EcalTrigTowerDetId::kSizeForDenseIndexing; iTT++) {
       EcalTrigTowerDetId ttid(EcalTrigTowerDetId::detIdFromDenseIndex(iTT));
-      unsigned iDCC(dccId(ttid) - 1);
+      // unsigned iDCC(dccId(ttid) - 1);
```
Comment (jfernan2) [inline DQM/EcalMonitorClient/src/TrigPrimClient.cc:109]: Better removed if not used

File: DQM/EcalMonitorTasks/src/RawDataTask.cc [hunk 3]
```diff
@@ -184,12 +170,10 @@ namespace ecaldqm {
       }
 
       if (feDesync > 0.) {
-        meDesyncByLumi.fill(dccId, feDesync);
+        // meDesyncByLumi.fill(dccId, feDesync);
```
Comment (jfernan2) [inline DQM/EcalMonitorTasks/src/RawDataTask.cc:173]: Better removed if not used

File: DQM/EcalMonitorTasks/src/TimingTask.cc [hunk 4]
```diff
@@ -123,7 +113,7 @@ namespace ecaldqm {
       if (energy > energyThreshold) {
         meTimeAll.fill(id, time);
         meTimeMap.fill(id, time);
-        meTimeMapByLS->fill(id, time);
+        // meTimeMapByLS->fill(id, time);
```
Comment (jfernan2) [inline DQM/EcalMonitorTasks/src/TimingTask.cc:116]: Better removed if not used

**Comment by tanmaymudholkar** [comment]
Hi @jfernan2 I have fixed those as well in the latest commit, sorry for missing them earlier.

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
This PR disables concurrent LS plots in  EcalDQMonitorTask and ESIntegrityTask 
Related to https://github.com/cms-sw/cmssw/issues/25090

**Comment by jfernan2** [comment]
@tanmaymudholkar  your changes not only remove the perLS plots but cause many plots to become empty and certification summaries which depend on them to become bad
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_11_2_X_2020-06-15-2300+1c6a6c/37048/dqm-histo-comparison-summary.html

**Comment by tanmaymudholkar** [comment]
@jfernan2 sorry for the delay, I am still looking into it... indeed the certification quality cuts have to be modified.

**Comment by jfernan2** [comment]
@tanmaymudholkar  I believe it depends on the time scale of your second PR, if you plan to introduce them back soon it makes no sense to go ahead with this; on the other side, if it is going to take longer, all your certification summaries wil be wrong all that long

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
what's the status/plan of this PR?

**Comment by jfernan2** [comment]
@silviodonato it depends on @tanmaymudholkar : the PR as it is now, only removes the conflicting code but this causes troubles in other parts of ECAL dqm

**Comment by silviodonato** [comment]
@tanmaymudholkar do you have any news?

**Comment by tanmaymudholkar** [comment]
@silviodonato sorry again for the delay, I was waiting for confirmation about the timeline for our next fix, but in any case I will post an update in a few hours.

**Comment by silviodonato** [comment]
FYI `CMSSW_11_2_0_pre3` will be cut tomorrow evening

**Comment by tanmaymudholkar** [comment]
Hi @silviodonato I tried to track down the remaining bad quality plots, but I think it would be best to proceed with the current PR. Some plots might be temporarily bad, but we are reintroducing the by-LS plots anyway.

**Comment by silviodonato** [comment]
@tanmaymudholkar does @cms-sw/dqm-l2 agree to go with the current PR?

**Comment by jfernan2** [comment]
I don't think this PR makes any help: the code is removed but a lot of plots are broken as mentioned before

**Comment by silviodonato** [comment]
@tanmaymudholkar I moved this PR as a "draft". Please open it back as soon as you have an update to be reviewed (please let me know if you do not have the permissions to do it)

**Comment by silviodonato** [comment]
moved to #31689

**Pull request closed without merging**